math/minuit2/inc/Minuit2/NumericalDerivator.h

```diff
@@ -0,0 +1,114 @@
+// @(#)root/mathcore:$Id$
+// Authors: L. Moneta, J.T. Offermann, E.G.P. Bos    2013-2018
+//
+/**********************************************************************
+ *                                                                    *
+ * Copyright (c) 2013 , LCG ROOT MathLib Team                         *
+ *                                                                    *
+ **********************************************************************/
+/*
+ * NumericalDerivator.h
+ *
+ *  Original version created on: Aug 14, 2013
+ *      Authors: L. Moneta, J. T. Offermann
+ *  Modified version created on: Sep 27, 2017
+ *      Author: E. G. P. Bos
+ */
+
+#ifndef ROOT_Minuit2_NumericalDerivator
+#define ROOT_Minuit2_NumericalDerivator
+
+#ifndef ROOT_Math_IFunctionfwd
+#include <Math/IFunctionfwd.h>
+#endif
```

Comment:
```suggestion
#include <Math/IFunctionfwd.h>
```
The `Math/IFunctionfwd.h` file already has the include guard in it. Why repeating it here?

Reply:
Yeah, that's so weird!